.github/workflows/dast.yml

```diff
@@ -0,0 +1,46 @@
+name: ZAP
+
+on:
+  push:
+    branches: [ story/77/dast/jorg3 ]
```

Comment:
This will change once it gets approved, it should be triggered when pushed to master. Other option can be having it run on every PR; thoughts?

Reply:
I think it should run on every PR and every push to `main`.

Reply:
Good call.